merlin-framework/src/main/java/gov/nasa/jpl/aerie/merlin/framework/annotations/ActivityType.java

```diff
@@ -25,4 +25,10 @@
   @interface EffectModel {
     Executor value() default Executor.Threaded;
   }
+
+  @Retention(RetentionPolicy.CLASS)
+  @Target(ElementType.METHOD)
+  @interface ControllableDuration {
+    String parameterName();
+  }
```

Comment:
Couldn't the annotation get the parameter name by looking at the parameter declared right below it? Annotations are attached to the line of code below them, aren't they? This is fine for now, but it may be an improvement to explore later if our users find it obnoxious to say the parameter name twice.

Reply:
I agree - the main reason I went with this approach is that we have several different ways to provide parameters (as records, as `@Parameter`, etc), and I wasn't sure how to handle all of them. I'll ping @pcrosemurgy to see if I can get some help on this.

Reply:
I think due to the lack of resources (such as time) we are fine to move forward with your approach, so I'm going to resolve this. After all, this will work just fine and isn't really cumbersome at all.